metrics.py

```diff
@@ -478,7 +551,9 @@ def ingest_dashboard(api):
 
     count = 0
     points = []
+    max_revision = 0
```

Comment:
Perhaps instead the revisions should be parsed by same bit of code since they are really rather similar (ie revision with this content for config) and need to be turned into points in the same way.

This also naturally works if further parsed files re migrated. The reason this was was setup with revision_at() and such is that in the future I wanted to parse the psudo-state ignored list to then create segments when the requests are ignored and filter them out of the various buckets like _backlog_. This would also be useful going forward since that information will likely also move.